client/src/main/java/com/vesoft/nebula/meta/client/MetaClientImpl.java

```diff
@@ -188,7 +188,7 @@ public Integer getEdgeType(String spaceName, String edgeName) {
         return getEdgeType(spaceNames.get(spaceName), edgeName);
     }
 
-    private void init() {
+    public void init() {
```

Comment:
I think there's no need to use init() cause it was already called in constructor

Reply:
We setting member variable in `Constructor`, another operation move into a method is better.